_rules/link-accessible-name-c487ae.md

```diff
@@ -254,7 +256,7 @@ Link is completely empty, but still shows up in focus order, so it should have a
 
 #### Failed Example 10
 
-`a` element where [accessible name][] through content is empty.
+This `a` element has an empty [accessible name][] from its content.
 
 ```html
 <a href="http://www.w3.org/WAI"> </a>
```

Comment:
This is essentially the same example as Failed Example 8. Do we need both?

Reply:
No, I don't think we do!